Isabelle Unicode for Bitbucket.user.js

```diff
@@ -560,7 +563,7 @@
 
         if (format.text_has_epilogue !== true) {
             // the 'normal' case
-            span = $(line).find(format.line_textspan);
+            span = $(line).find(format.line_textspan).addBack(format.line_textspan);
```

Comment:
is it worth a comment what this now does, assuming any of us have any idea?

Reply:
It's including the previous element (which is `line`) when trying to find an instance of `format.line_textspan`. I added it to handle the new case where `code_line` and `line_textspan` are the same element.
There's probably a nicer way to do this but it was the simplest change that I could find with my extremely incomplete jquery knowledge.

Reply:
*blink*
Well done figuring that out. I can't contribute anything better.